js/sum.js

```diff
@@ -0,0 +1,46 @@
+let addBtn = document.getElementById('add-btn');
+let newNum = document.getElementById('new-num');
+let numsContainer = document.getElementById('added-nums-container');
+let sumElem = document.getElementById('sum');
+let isOdd = false;
+
+addBtn.addEventListener('click', () => {
+    if (newNum.value === '') {
```

Comment:
You are counting on the ui for the validation. This is not a good practice. And it's not very obvious. If some reason the input's type will be changed to 'text' this validation will not work and let strings trough.
And also I still can write something like "555e" into the input itself.

Reply:
Fixed :)